packages/commerce-sdk-react/src/hooks/ShopperBaskets/useShopperBasketPaymentMethods.ts

```diff
@@ -0,0 +1,16 @@
+import {PaymentMethodResult} from 'commerce-sdk-isomorphic'
+import {QueryResponse} from '../../types'
+import {ShopperBasketParams} from './types'
+
+const useShopperBasketPaymentMethods = (
+    params: ShopperBasketParams,
+    source: []
+): QueryResponse<PaymentMethodResult> => {
+    return {
+        data: {},
+        isLoading: true,
+        error: undefined,
+    }
+}
+
+export default useShopperBasketPaymentMethods
```

Comment:
What would it look like for users to import this hook? 🤔 

`import useShopperBasketPaymentMethods from 'commerce-sdk-react/src/hooks/ShopperBaskets/useShopperBasketPaymentMethods'`

Reply:
What if the import statement looks like this instead?

`import {useShopperBasketPaymentMethods} from 'commerce-sdk-react/src/hooks/ShopperBaskets'`

I think doing it this way would be:
- shorter to type
- quicker to type because of the auto-completion ← named vs default export
- less files → does each hook have to be in its own file?

Reply:
There's also the option of declaring our exports in the package.json so that import paths are nicer.

Reply:
I'd imagine it will look like `import useShopperBasketPaymentMethods from 'commerce-sdk-react`!